lib/bambora/v1/profile_resource.rb

```diff
@@ -100,6 +100,54 @@ def get(customer_code:)
         client.get(path: "#{sub_path}/#{customer_code}", api_key: api_key)
       end
 
+      # Make a PUT Request.
```

Comment:
This method documentation doesn't seem to match the method. Can this be updated?

Reply:
You're correct. I updated the example below.

Reply:
Good catch. I updated it so it matches.